fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/MessageValidator.java

```diff
@@ -26,6 +26,12 @@ public static MessageValidatorBuilder verify() {
 
   public static class MessageValidatorBuilder {
 
+    // Defines how old messages can be to be considered valid, keeping it non-restrictive here for
+    // now
+    public static final long VALID_DELAY = 100000000;
+    private final long CURRENT_TIME = Instant.now().getEpochSecond();
```

Comment:
While this is not problematic right now this could lead to unintuitive behaviours in the future if people use this class without realising it was designed so. To be more defensive, it would be bette to define `validPastTime` as a method so that `currentTime` is always correct